scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,138 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
+    RETURNS VOID AS $$
+DECLARE
+    cdb_group_role TEXT;
+BEGIN
+    cdb_group_role := cartodb._CDB_Group_GroupRole(group_name);
+    IF NOT EXISTS ( SELECT 1 FROM pg_roles WHERE rolname = cdb_group_role )
+    THEN
+        EXECUTE 'CREATE ROLE "' || cdb_group_role || '" NOLOGIN;';
```

Comment:
I think it would be better to have some kind of "migration" and when you install the new extension the role is created. It could be really weird in the future trying to use that role and it not being there. 

I'm sure this can be done, let's wait for @pramsey cc @rochoa